.github/workflows/ci.yml

```diff
@@ -72,12 +73,12 @@ jobs:
       - run: pip3 install nox packaging
       - run: nox -s build_docs
 
-  Run-Unit-Tests:
+  Run-Unit-Tests:  # yamllint disable-line
     strategy:
       fail-fast: false
       matrix:
-        python: [ '3.8', '3.9', '3.10' ]
-        airflow: [ 2.5 ]
+        python: ['3.8', '3.9', '3.10']
```

Comment:
We should add 3.11 but that can happen in a Separate PR

Reply:
Created ticket to track this: https://github.com/astronomer/astro-provider-databricks/issues/67